client/spec/multicontent_spec.js

```diff
@@ -6,13 +6,14 @@ var workspace = require('./helpers/workspace'),
     content = require('./helpers/content'),
     multicontent = require('./helpers/multicontent');
 
-describe('multi content view', function() {
+// todo(petr): those features should go to list
+xdescribe('multi content view', function() {
```

Comment:
Is "moving those features to list" (whatever that means) doable within this PR? If yes, we should do it in order to avoid having a disabled test lying around (these tend to get forgotten).